modules/compiler/vecz/tools/source/veczc.cpp

```diff
@@ -372,10 +378,15 @@ int main(const int argc, const char *const argv[]) {
       [&] { return vecz::TargetInfoAnalysis(TICallback); });
   passMach.getMAM().registerPass(
       [&] { return compiler::utils::BuiltinInfoAnalysis(); });
+  passMach.getMAM().registerPass(
+      [&] { return compiler::utils::SubgroupAnalysis(); });
   passMach.getFAM().registerPass([] { return llvm::TargetIRAnalysis(); });
   passMach.getMAM().registerPass([] {
     compiler::utils::DeviceInfo Info{/*half*/ 0, /*float*/ 0, DoubleSupport,
                                      /*MaxWorthWidth*/ 64};
+    for (const auto S : SGSizes) {
+      Info.reqd_sub_group_sizes.push_back(S);
```

Comment:
I'm wondering whether this is a good name, given there is the ability to suggest a sub group size in the language? Maybe supported_sub_group_sizes?